webapp/src/components/header.tsx

```diff
@@ -110,7 +108,7 @@ function Header() {
             <Container maxWidth={false}>
                 <Toolbar disableGutters>
                     {logoAndTitle()}
-                    <SearchBar/>
```

Comment:
What is the reason for removing it?
Should we also remove `search_bar.tsx` file

Reply:
Search is not working, for now. It's easier to remove for consistency. Will be added later with full functionality. So `search_bar.tsx` is needed.